packages/x-components/src/components/modals/base-events-modal.vue

```diff
@@ -4,6 +4,7 @@
     @focusin:body="emitBodyClickEvent"
     :animation="animation"
     :open="isOpen"
+    v-bind="$attrs"
```

Comment:
I am not sure if this is right, I don't like to pass the same prop between multiple components to send it to the nested one.

Reply:
I agree but I don't see another way apart from injecting it.